code/game/jobs/job/security.dm

```diff
@@ -43,6 +43,7 @@
 		var/obj/item/weapon/implant/loyalty/L = new/obj/item/weapon/implant/loyalty(H)
 		L.imp_in = H
 		L.implanted = 1
+		H.sec_hud_set_implants()
```

Comment:
you may want to move this to the very button, underneath L.part = affected, just to make sure all implant stuff is done.